scripts/funcotator/data_sources/gnomAD/liftoverVcf.wdl

```diff
@@ -0,0 +1,373 @@
+# Lifts over a VCF file from one reference version to another.
```

Comment:
Is this script supported?

Reply:
I misunderstood what supported/unsupported means in this context.

No, none of these need to be supported.